### ISSUE TITLE
Wrapping fails in files with tab-based indentation

### ISSUE THREAD

**Issue by abastardi** [stkb/Rewrap#299]
In files with tab-based indentation, comments are wrapped, but (a) the comment symbols are not stripped from lines that are combined, and (b) comment symbols are not added to new lines that are created. For example, the following:

```javascript
	// comment line 1
	// comment line 2
	// comment line 3
	// comment line 4
	// comment line 5
```

is transformed to:

```javascript
	// comment line 1 // comment line 2 // comment line 3 // comment line 4 //
	comment line 5
```

**Comment by stkb** [comment]
Hi, tabs are supported and it's still working fine for me. What type of file is this you're working in?

**Comment by abastardi** [comment]
Sorry, I just realized this only happens in Svelte (`.svelte`) files. It should work if Rewrap were to treat `.svelte` files as `.html` files. Is there any way to make that happen?

**Comment by stkb** [comment]
Thanks. 

I wasn't sure if this would maybe go in the next major version, but that's going to be a little while yet, so I've released it in v1.15.4.

**Comment by abastardi** [comment]
It works. Thanks a lot.
